zul/src/archive/web/js/zul/inp/less/slider.less

```diff
@@ -29,6 +30,30 @@
 		background-color: @sliderAreaBackgroundColor;
 	}
 
+	&-knob-area {
+		stroke: @sliderAreaBackgroundColor;
+		data-angleOffset: 0;
+	}
+
+	&-knob-inner {
+		stroke: @sliderBackgroundColor;
+		data-angleOffset: 0;
+	}
+
+	&-knob-svg {
+		cursor: pointer;
+	}
+
+	&-input {
+		border: 1px solid @sliderAreaBackgroundColor;
+		background: none;
+		-webkit-appearance: none;
+		color: rgb(135, 206, 235);
```

Comment:
Why this color, can it be theme-related variable? Like @textColorDefault

Reply:
ok